src/deploy/index.js

```diff
@@ -107,8 +107,12 @@ var deploy = function (targetNames, options, customContext = {}) {
         _.each(context.hosting.deploys, function (deploy) {
           logger.info(clc.bold("Hosting URL:"), utils.addSubdomain(api.hostingOrigin, deploy.site));
         });
-        const versionNames = context.hosting.deploys.map((deploy) => deploy.version);
-        return { hosting: versionNames.length === 1 ? versionNames[0] : versionNames };
+        var siteDetails = {};
```

Comment:
```suggestion
        const siteDetails = {};
```

Reply:
I had `const` initially, but then noticed that this `js` file is using `var` everywhere else. I'd like to leave it as `var` if it is okay with you.